src/apiFetch.jsx

```diff
@@ -0,0 +1,31 @@
+// Fetches API data
+
+const API_KEY = '06865a41347d5e83bed0257e5d40c860';
```

Comment:
you shouldn't write the api key directly in the code, the same mistake we've done :)
we should've used .env file instead, and git ignore the .env file to avoid pushing it to Github
but since our api key is free, it won't be a big deal to have it in the code pushed to github